nwb_conversion_tools/utils/si090nwbephyswriter.py

```diff
@@ -128,11 +128,19 @@ def _get_unit_spike_train_times(self, unit_id, segment_index=0):
         return self._get_unit_spike_train_ids(unit_id, segment_index)/self._get_unit_sampling_frequency()
 
     def _get_unit_property_names(self):
-        return list(self.sorting._properties.keys())
+        if 'max_channel' not in self.sorting._properties:
+            prop_names = list(self.sorting._properties.keys()).extend('max_channel')
+        else:
+            prop_names = list(self.sorting._properties.keys())
```

Comment:
`self.sorting.get_property_keys()` instead of `self.sorting._properties`

Reply:
yeah, done